wwwroot/index.html

```diff
@@ -1,469 +1,855 @@
+            </p>
+          </div>
+        </label>
+      </div>
+    </section>
+    <section class="section-2">
+      <img class="section-2-bg" src="images/section-2-bg-2.svg" alt="">
+
+      <div class="specific-features">
+
+        <input type="radio" name="option" id="option1" class="radio-input" checked>
+        <input type="radio" name="option" id="option2" class="radio-input">
+        <input type="radio" name="option" id="option3" class="radio-input">
+        <input type="radio" name="option" id="option4" class="radio-input">
+        <input type="radio" name="option" id="option5" class="radio-input">
+        <input type="radio" name="option" id="option6" class="radio-input">
+        <input type="radio" name="option" id="option7" class="radio-input">
+        <input type="radio" name="option" id="option8" class="radio-input">
+        <div class="text-column">
+          <div class="option-text  text-option1">
+            <h6 class="option-label">Tor Everywhere</h6>
+            <p class="option-description">All your traffic is routed through the <a href="https://www.torproject.org/">Tor Network,</a> ensuring complete privacy and
+              anonymity.
+              <hr style="margin-top: 20px; border: none;">Wasabi's custom Tor integration ensures that no two important requests appear to originate from the same source, making it virtually impossible for
+              anyone to trace your activity. 
+              <hr style="margin-top: 20px; border: none;">Whether you're sending transactions or synchronizing with the blockchain, your privacy is always
+              protected.</p>
+          </div>
+          <div class="option-text  text-option2">
+            <h6 class="option-label">Private Blockchain Synchronization</h6>
+            <p class="option-description">Wasabi uses Compact Filters to synchronize with the blockchain privately. 
+              <hr style="margin-top: 20px; border: none;">Instead of relying on a central
+              server or having to download the entire blockchain, Wasabi requests only the relevant blocks for your transactions. 
+              <hr style="margin-top: 20px; border: none;">This ensures that noone can determine how much Bitcoin you own or which transactions belong to you.</p>
+          </div>
+          <div class="option-text  text-option3">
+            <h6 class="option-label">Multi-Party Trasactions</h6>
+            <p class="option-description" style="display: flex; flex-direction: column; gap: 20px;">
+              Wasabi integrates the WabiSabi Trustless Multi-Party Transaction Engine, a cutting-edge protocol for collaborative
+                transactions.
+                
+                <hr style="margin-top: 20px; border: none;">This innovative feature offers several advantages:
+
+            <ul>
+              <li><b>Hide Payment Origins:</b> Conceal the inputs used for payments from recipients and observers.</li>

```

Comment:
```suggestion
              <li><b>Hide Payment Origins:</b> conceal the inputs used for payments from recipients and observers.</li>

```
I think it's incorrect English to start with a capital there